### PR TITLE
Update transfer funds modal styles and add new component

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
## Description

This PR updates styles of the TokensMoveDialog and adds a new field to it.

**New stuff** ✨

* Optional text area on the transfer funds dialog to explain the reason for the transfer.

**Changes** 🏗

* Moved MoveTokensDialog to `dashboard` module
* Changed MoveTokensDialog name to TransferFundsDialog
* Updated styles of from/to fields so that they are in line with each other
* Changed MovedTokensDialog title to "Transfer Funds"

Resolves DEV-18

FireShot Capture 006 - Colony - localhost

### CODE REVIEW

File: src/modules/admin/components/Tokens/Tokens.tsx
Comment (alicjakujawa) [inline src/modules/admin/components/Tokens/Tokens.tsx:22]:
```suggestion
'~dashboard/TransferFundsDialog'```
Should work fine

File: src/modules/admin/components/Tokens/index.ts
Comment (alicjakujawa) [inline src/modules/admin/components/Tokens/index.ts:1]: Not sure why we exporting it here 🤔  @rdig What was the reason for those exports ealier?

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx
Comment (alicjakujawa) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:172]: if we returning just one value, should we just return from? and replace 
const [fromDomainTokenBalance]= with just 
const fromDomainTokenBalance=

**Comment by alicjakujawa** [comment]
Tests locally, there are some minor difference between design and implementation i noticed:
1 - the line, which is below "Amount" should be a bit higher:
Screenshot 2020-11-26 at 10 35 45
2 - Label text should be in different color, should be fine if you change to:
              appearance={{
                    colorSchema: 'grey',
                  }}
3. Top padding of modal should be bigger, but i added this change in my PR, so after it will be merge we should apply this to all modals, as it seems to be change for all of them. (36px padding vs current 30px)

**Review by alicjakujawa — CHANGES_REQUESTED** [review]
Minor things/questions to change 🔥

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:245]: In general all the fonts on this modal are too small. Please follow the spec on figma for proper font sizing.

For example, in this case:
Screenshot from 2020-11-26 14-43-34

Your
Screenshot from 2020-11-26 14-43-53
 font size is `12px` while the spec requires it to be `13px`

**Review by rdig — CHANGES_REQUESTED** [review]
Overall a decent first issue. I've noted the things I've picked up in terms of specs mis-alignments.

Please note that in the case of namespacing, while I just noted one of the entries, the change needs to be applied globally to the file.

:+1:

**Comment by rdig** [comment]
@alicjakujawa @ArmandoGraterol regarding:

> 1 - the line, which is below "Amount" should be a bit higher:

No, leave it like it is. I consider the spec to be wrong in this case

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:322]: Please add the `grey` color schma here:
```suggestion
          appearance={{ theme: 'fat', resizable: 'vertical', colorSchema: 'grey' }}
```

Comment (rdig) [reply src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:322]: Also do the same for the other input on the page :+1:

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css
Comment (alicjakujawa) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css:42]: we have var for it -  --size-medium-l 🙈

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css
Comment (alicjakujawa) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css:50]: --size-smallish

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:274]: titles need to be message descriptors as well

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:294]: Sorry if I made myself misunderstood, but this should have just been applied to the two select components and the one textarea.

The way you implemented it initially was correct:

Screenshot from 2020-11-27 14-30-47

Just remove the `colorSchema` from here, and we're all good.

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css:36]: While this is technically possible currently via our webpack loaders, future updates might break it.

Please move it to a new class declaration. Something like:
```js
.transferIcon svg {
  fill: var(--temp-grey-blue-7);
}
```

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:335]: Please add a character limit of `4000` to this textarea, sorry I missed it the first time around.

You can do this by passing the `maxLength` prop.

Eg:
```
maxLength={4000}
```

**Review by rdig — CHANGES_REQUESTED** [review]
Overall pretty good, you fixed most of the issues found in the first round of reviews.

I've found some other minor discrepancies plus there are Alicja's notes.

Once you change those, this will be good to go.

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css:18]: `var(--weight-bold)`

File: src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx
Comment (rdig) [inline src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx:263]: The default value could have been put inside the `useMemo` hook, but I guess this is fine as well

**Review by rdig — APPROVED** [review]
This looks good.

I found another place where you need to put a variable, but other than that this is good to go.

I'm pre-approving this, so after you fix the variable, you can merge it.

Congrats on your first PR! :100:

**Pull request merged**
